### PR TITLE
Improve layout

### PR DESCRIPTION
**Pull request by andysellick**
## What / why
Visual and layout improvements to the implementation record, based partly on feedback from recent discussions about it.

## Visual changes
Many and varied.

New homepage:
Screenshot 2023-03-30 at 08 24 58

New menu/layout example:
Screenshot 2023-03-30 at 08 25 08

Syntax highlighting:
Screenshot 2023-03-30 at 08 25 24

### CODE REVIEW

File: docs/_sass/application.scss [hunk 1]
```diff
@@ -82,3 +83,21 @@ $govuk-assets-path: "/govuk-google-analytics/assets/";
...
   @include govuk-font(19);
 }
+
+.highlight-json {
+  padding: govuk-spacing(2);
+  border: solid 1px $govuk-border-colour;
+  background: govuk-colour("light-grey");
+}
+
+.highlight-json__brace {
```
Comment (AshGDS) [inline docs/_sass/application.scss]: Empty class!

File: docs/index.html [hunk 2]
```diff
@@ -2,4 +2,22 @@
...
 <h1 class="govuk-heading-l">{{ page.title }}</h1>
-<p class="govuk-body">This site contains a record of the data implementation for Google Analytics 4 on GOV.UK.</p>
+<p class="govuk-body">This site contains a record of the data implementation for Google Analytics 4 on GOV.UK Publishing. It has been built as an aid to developers on GOV.UK Publishing but is being made available for anyone who may find our approach useful.</p>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <h2 class="govuk-heading-l govuk-!-font-weight-bold">Data</h2>
+    <p class="govuk-body">Details of the data pushed to the dataLayer when tracking occurs, including event and attribute details.</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">
+      <a href="{{ site.baseurl }}/data" class="govuk-link govuk-!-font-weight-bold">Browse data</a>
```
Comment (AshGDS) [inline docs/index.html]: This link is broken for me

File: docs/index.html [hunk 3]
```diff
@@ -2,4 +2,22 @@
...
+    </p>
+  </div>
+
+  <div class="govuk-grid-column-one-half">
+    <h2 class="govuk-heading-l govuk-!-font-weight-bold">Documentation</h2>
+    <p class="govuk-body">Relevant documentation including our approach.</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">
+      <a href="{{ site.baseurl }}/documentation" class="govuk-link govuk-!-font-weight-bold">Browse documentation</a>
```
Comment (AshGDS) [inline docs/index.html]: This link is broken for me

**Review by AshGDS — CHANGES_REQUESTED** [review]
Looks good, a few small changes to make :+1:

File: docs/_sass/application.scss [hunk 4]
```diff
@@ -82,3 +83,21 @@ $govuk-assets-path: "/govuk-google-analytics/assets/";
...
+
+}
+
+.highlight-json__key {
+  color: govuk-colour("dark-blue");
+}
+
+.highlight-json__value {
+  color: govuk-colour("red");
```
Comment (AshGDS) [inline docs/_sass/application.scss]: Just noticed as well, this doesn't meet our accessibility requirements for colour contrast against the grey background used.

**Review by AshGDS — CHANGES_REQUESTED** [review]

File: docs/_sass/application.scss (hunk 1 above)
Comment (andysellick) [reply docs/_sass/application.scss]: Thanks, have removed (and references to it).

File: docs/index.html (hunk 2 above)
Comment (andysellick) [reply docs/index.html]: Ah yes, forgot to update when I restructured the file layout. Fixed now.

File: docs/index.html (hunk 3 above)
Comment (andysellick) [reply docs/index.html]: Also fixed.

File: docs/_sass/application.scss (hunk 4 above)
Comment (andysellick) [reply docs/_sass/application.scss]: Was relying on the DS colours but have chosen some others to meet contrast.

**Review by AshGDS — APPROVED** [review]
All good :+1:

File: docs/_sass/application.scss (hunk 4 above)
Comment (AshGDS) [reply docs/_sass/application.scss]: No worries - yeah I think that colour is fine when on a white background, but the grey made it inaccessible.

**Pull request merged**